doc/source/whatsnew/v1.0.0.rst

```diff
@@ -1024,6 +1024,7 @@ Interval
 Indexing
 ^^^^^^^^
 
+- List datetime string indexer could fail on :class:`PeriodIndex` or :class:`DatetimeIndex` (:issue:`11278`)
```

Comment:
can you make this a bit more user friendly, not really sure what has changed on first read